functions/api/action.js

```diff
@@ -265,6 +265,43 @@ router.post('/moveProduct', async (req, res, next) => {
     res.status(201).send({ fromBranchTransactionId, toBranchTransactionId })
 })
 
+/**
+ * Route to reject product by headoffice
+ * @returns 200
+ */
+router.post('/rejectProduct', async (req, res, next) => {
```

Comment:
reject is for rejecting a transaction, right? Should it be rejectTransaction?  
Also, can the user who created a request, cancel the request?

Reply:
but by this time, transactions will not be created. I'll change to rejectRequest. Umm, may be good to have feature. But for now, only headoffice can accept or reject
Are you asking because, we need to add isAdmin check?

Reply:
That is one thing. But what if user raises a request for wrong product or with wrong quantity?

Reply:
PO should give me answer for this q. Asked sunitha :)